azure/packages/azure-client/src/AzureFunctionTokenProvider.ts

```diff
@@ -12,6 +12,11 @@ import { AzureMember } from "./interfaces";
  * Token Provider implementation for connecting to an Azure Function endpoint for
  * Azure Fluid Relay token resolution.
  */
+/**
+ * @deprecated 1.1.2, This API will be removed in 1.1.3
+ * No replacement since it is not expected anyone will use this token provider as is
```

Comment:
We will also need to add notes to `BREAKING.md` explaining why this was removed and what users need to do in preparation for its removal. The contents probably look very similar to this comment :)